wire-ios-data-model/Source/Public/TextMessage.swift

```diff
@@ -0,0 +1,75 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireLinkPreview
+
+@objc(ZMTextMessage) @objcMembers
+final class TextMessage: ZMMessage, TextMessageData {
```

Comment:
Do you know if we need to change the class name to `TextMessage` in the Core Data entity inspector?

Reply:
We don't have to change it, what matters is the identifier put into `@objc`.